pkg/asset/machines/baremetal/hosts.go

```diff
@@ -141,6 +141,12 @@ func Hosts(config *types.InstallConfig, machines []machineapi.Machine) (*HostSet
 			// with a machine without triggering provisioning. We only
```

Comment:
Should we clean up the comment about "without setting Image", since afaict that is irrelevant?

Reply:
I'll see if someone can lgtm the change right now and will either fix it right here or follow-up.